src/theme/components/searchBar.ts

```diff
@@ -10,14 +10,34 @@ const SearchBar = defineMultiStyleConfig({
       base: "xs",
       md: "auto",
     },
-    flexFlow: {
-      base: "column nowrap",
-      md: "row nowrap",
-    },
     ".textInput": {
-      flexGrow: "1",
+      flexGrow: 1,
+      "div > input": {
+        borderRightRadius: 0,
+      },
+    },
+    flexFlow: { base: "column nowrap", md: "row" },
+    ".searchButton": {
+      minWidth: "44px",
+      maxWidth: "80px",
```

Comment:
This should be removed.  This is what is causing the button spacing to get out of whack.

Reply:
On >md breakpoints it seems like this is needed though?